shelf_router/lib/src/router.dart

```diff
@@ -135,24 +134,24 @@ class Router {
   }
 
   /// Get a [Handler] that will route incoming requests to registered handlers.
-  Handler get handler {
+  Handler get handler => call;
```

Comment:
Should we consider marking this as `@deprecated`? Advocating that the `Router` class is an `Handler` on its own?

Reply:
Yes, I can do that.